EventFilter/Phase2TrackerRawToDigi/interface/Phase2TrackerSpecifications.h

```diff
@@ -26,7 +26,7 @@ namespace Phase2TrackerSpecifications
     static const int CIC_Z_BOUNDARY_PIXEL = 15;
     static const int CIC_Z_BOUNDARY_STRIPS = 0;
 
-    static const int MAX_DTC_ID = 216;
+    static const int MAX_DTC_ID = 217;
```

Comment:
This is wierd. Since there are 216 DTCs in the Tracker, why does the ID run from 1 to 217?

Reply:
we could choose to loop from 0 to 216 or from 1 to 217 to run over all the DTCs, as the various loops are currently from MIN_DTC_ID to < MAX_DTC_ID. Or change "<" to "<=" in the loops. I can change according to the general preference.

Reply:
Why can't we determine the number of DTCs from the Tracker cabling map?
I would avoid 217. So prefer "<=" if that allows you to use 216.

Reply:
changed